internal/command/plan_test.go

```diff
@@ -1,21 +1,92 @@
-package command_test
+package command
```

Comment:
I've moved this into the `command` package so we can get rid of `command.SetPlanWriter()` and not introduce more exported methods that are only used for testing.